### PR TITLE
ACMS-825: Added template for maintenance and update php page.

### PR DESCRIPTION
**Pull request by panshulK**
**Motivation**

Fixes #[ACMS-812](https://backlog.acquia.com/browse/ACMS-812)

Create maintenance page template and do some design so that all these page look similar in design:

- Install page
- Update page
- Maintenance page

**Proposed changes**

Provide brand new experience for acquia cms maintenance and update php page.

**Testing steps**

- Install site check newly created install page for acquia cms distribution.
- Install site, put site on maintenance mode and check newly created maintenance page for acquia cms distribution.
- Visit `/update.php` to check newly created update.php page experience.

**Merge requirements**
- [_Enhancement_] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

File: tests/src/ExistingSiteJavascript/UpdatePageTest.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+  /**
+   * Tests update page design.
+   *
+   * @throws \Behat\Mink\Exception\ElementNotFoundException
+   * @throws \Behat\Mink\Exception\ResponseTextException
+   * @throws \Drupal\Core\Entity\EntityStorageException
+   */
+  public function testUpdatePage() {
+    $account = $this->createUser();
```
Comment (vishalkhode1) [inline tests/src/ExistingSiteJavascript/UpdatePageTest.php:25]: @chandan-singh7929 Not sure if we need to delete the user & role both or does it handle that automatically ?

File: acquia_cms.profile [hunk 2]
```diff
@@ -330,3 +330,21 @@ function acquia_cms_update_8001() {
...
+ *
+ * Default template: maintenance-page.html.twig.
+ *
+ * @param array $variables
+ *   An associative array containing:
+ *   - content - An array of page content.
+ *
+ * @see template_preprocess_maintenance_page()
+ */
+function cohesion_theme_preprocess_maintenance_page(array &$variables) {
```
Comment (vishalkhode1) [inline acquia_cms.profile:345]: @chandan-singh7929 Why this hook is cohesion_theme. This should be `acquia_cms`. Right ?

**Review by vishalkhode1 — CHANGES_REQUESTED** [review]

File: acquia_cms.profile (hunk 2 above)
Comment (chandan-singh7929) [reply acquia_cms.profile]: Done

File: tests/src/ExistingSiteJavascript/UpdatePageTest.php (hunk 1 above)
Comment (chandan-singh7929) [reply tests/src/ExistingSiteJavascript/UpdatePageTest.php:25]: We don’t have to do that, it automatically handles that.

File: themes/acquia_claro/acquia_claro.theme [hunk 3]
```diff
@@ -16,13 +16,13 @@ use Drupal\acquia_claro\PreRender;
  *   An associative array containing:
  *   - content - An array of page content.
  *
- * @see template_preprocess_maintenance_page()
+ * @see template_preprocess_install_page()
  */
 function acquia_claro_preprocess_install_page(array &$variables) {
```
[the remark this answers is not in the document]
Comment (chandan-singh7929) [reply themes/acquia_claro/acquia_claro.theme]: Done

**Review by vishalkhode1 — APPROVED** [review]

**Pull request merged**